### DISCUSSION TITLE
Has Canopy Project been abandoned?

### DISCUSSION THREAD

**Discussion by avibathula** [pinecone-io/canopy#339] [General]
I do not see any pull-requests being reviewed or accepted. And no commits since 0.9 release.

**Comment by miararoy** [comment]
Hi Folks, Canopy project have not been abandoned. The team behind Canopy is working on something big in this space, soon to be announced. We will also have some updates on canopy, including bug fixes and security features. Stay tuned!

**Comment by salmanjamali** [comment]
@miararoy Is this still the plan or has anything changed?

**Reply by avibathula** [reply]
Anything that is dormant for over 4-5 months is as good as gone Zzzzzz. Given how rapidly world is changing, losing customer (dev) attention for that long is eternity and people move on to something else and slowly stickiness forms making it hard to move/migrate and thus putting the final nail in the coffin.

Just my 2cents.

**Reply by miararoy** [reply]
We have added Robust Intelligence on PR # and have released https://www.pinecone.io/blog/pinecone-assistant/ which we currently migrating Canopy users to. All comments true, we will be updating on the project soon. 🙏

**Reply by salmanjamali** [reply]
Appreciate the update.
